playground/lgsvl/executeRandomTests/main.py

```diff
@@ -0,0 +1,110 @@
+from queue import Queue
+
+from lgsvl import Simulator
+from shapely.geometry import LineString, Point
+from shapely.ops import unary_union
+
+sim = None
+roads = None
+road_points = None
+junctions = None
+allowed_area = None
+entry_segment = None
+final_segment = None
+ego = None
+
+
+def calc_allowed_area():
+    global road_points, entry_segment, final_segment, junctions
```

Comment:
Here you rely on global variables. But are those needed? I mean, the main loop should work with instance variables not global ones. Using global variable usually complicates things when one want to change the code.

I hoped that given an input like map/xml and start-end points you could re-compute this information per each test.

Reply:
Some of them are needed. I actually just copied most of them to make them available. It's not a product.
If you look closely I do recompute this information for each connection (i.e. start-end point).